ckan/model/group.py

```diff
@@ -104,11 +104,11 @@ def related_packages(self):
     def __unicode__(self):
         # refer to objects by name, not ID, to help debugging
         if self.table_name == 'package':
-            table_info = 'package=%s' % meta.Session.query(_package.Package).\
-                get(self.table_id).name
+            pkg = meta.Session.query(_package.Package).get(self.table_id)
+            table_info = 'package=%s' % pkg.name if pkg else 'None'
```

Comment:
In the thick of a purge, one of my tests caused an exception here. The Package was purged, yet in the bowels of SQLAlchemy it was doing str(member), so `meta.Session.query(_package.Package).get(self.table_id)` was None and therefore calling `.name` on it caused an Exception. So this just adds an 'if' to cope with that situation.